vscode/src/documentation.ts

```diff
@@ -27,20 +27,24 @@ export async function showDocumentationCommand(extensionUri: Uri) {
   const worker = getCompilerWorker(compilerWorkerScriptPath);
   const docFiles = await worker.getDocumentation(program.programConfig);
 
-  const documentation: string[] = [];
+  const documentation: IDocFile[] = [];
   for (const file of docFiles) {
     // Some files may contain information other than documentation
     // For example, table of content is a separate file in a special format
     // We check presence of qsharp.name in metadata to make sure we take
     // only files that contain documentation from some qsharp object.
     if (file.metadata.indexOf("qsharp.name:") >= 0) {
-      documentation.push(file.contents);
+      documentation.push(file);
     }
   }
 
+  let projectName = program.programConfig.projectName ?? "App";
```

Comment:
This shouldn't need a fallback since it's a `string` (not null).

Reply:
Done